src/spec/integration/import.spec.coffee

```diff
@@ -116,11 +116,11 @@ describe 'Import integration test', ->
         done()
       .catch (err) -> done _.prettify(err)
 
-    it 'should import a simple product', (done) ->
+    it 'should import a simple product, without setting state', (done) ->
       csv =
         """
-        productType,name,variantId,slug,key,variantKey
-        #{@productType.id},#{@newProductName},1,#{@newProductSlug},productKey,variantKey
+        productType,name,variantId,slug,key,variantKey,state
```

Comment:
do we have to provide state column every time? It should work even if we don't provide it, right?

Reply:
Done.